chia/full_node/mempool_manager.py

```diff
@@ -874,6 +875,8 @@ async def local_get_coin_records(names: Collection[bytes32]) -> list[CoinRecord]
             f"minimum fee rate (in FPC) to get in for 5M cost tx: {self.mempool.get_min_fee_rate(5000000)}"
         )
         self.mempool.fee_estimator.new_block(FeeBlockInfo(new_peak.height, included_items))
+        duration = time.monotonic() - new_peak_start
+        log.log(logging.WARNING if duration > 1 else logging.INFO, "new_peak() took {duration:0.2f} seconds")
```

Comment:
or pass `duration` separately with `%` formatting

```suggestion
        log.log(logging.WARNING if duration > 1 else logging.INFO, f"new_peak() took {duration:0.2f} seconds")
```